app/pyproject.toml

```diff
@@ -4,49 +4,66 @@ version = "0.1.0"
 description = "A template flask API for building ontop of"
 packages = [{ include = "api" }]
 authors = ["Nava Engineering <engineering@navapbc.com>"]
+include = ["openapi.yml"]
 
+# Base packages for all versions of the images
+# Any packages needed for scripts called from outside of the app
+# would go here, ie create-user-csv
 [tool.poetry.dependencies]
 python = "^3.10"
-SQLAlchemy = {extras = ["mypy"], version = "^1.4.40"}
-alembic = "^1.8.1"
-psycopg2-binary = "^2.9.3"
-python-dotenv = "^0.20.0"
-pydantic = "^1.10.0"
-botocore = "^1.27.67"
-boto3 = "~1.24.67"
-smart-open = "^6.1.0"
-pytz = "^2022.2.1"
-APIFlask = "^1.1.3"
-marshmallow-dataclass = {extras = ["enum", "union"], version = "^8.5.8"}
-marshmallow = "^3.18.0"
-
-[tool.poetry.dev-dependencies]
-pytest = "^6.0.0"
-black = "^22.6.0"
-flake8 = "^5.0.4"
-flake8-bugbear = "^22.8.23"
-flake8-alfred = "^1.1.1"
-isort = "^5.10.1"
-mypy = "^0.971"
-moto = {extras = ["s3"], version = "^4.0.2"}
-types-pytz = "^2022.2.1"
-coverage = "^6.4.4"
-Faker = "^14.2.0"
-factory-boy = "^3.2.1"
-bandit = "^1.7.4"
 
+[tool.poetry.group.app]
```

Comment:
In the NJ repo, we have the three groups so that we can run a python script to build the ECS config JSON that doesn't have the app or dev packages. We decided to do this since that script doesn't need anything special, but doesn't need the extra overhead of the other packages. I guess a better thing would to have a fourth group for something like this, called `scripts`, and add that stage to Dockerfile + the compose file?

Reply:
hm, that need seems awfully specific to that project so let's not carry that pattern/idea over here. let's default to the simplest thing that most projects will need which is just default prod dependencies + a dev group. i don't see a need to dynamically generate the ECS config JSON.